README.md

```diff
@@ -1470,6 +1474,51 @@ scanId, err := xrayManager.ScanGraph(graphScanParams)
 // scanId should be received from xrayManager.ScanGraph(graphScanParams) request.
 scanResults, err := xrayManager.GetScanGraphResults(scanId)
 ```
+
+#### Generate Vulnerabilities Report
+```go
+reportRequest := services.ReportRequestParams{
+  Name: "example-report",
+  Filters: services.Filter{
+    HasRemediation: true,
+    Severity:       []string{ "High" },
+  },
+  Resources: services.Resource{
+    IncludePathPatterns: []string{ "/example-sub-dir/**" },
+    Repositories: []services.Repository{
+      {
+        Name: "example-repository",
+      },
+    },
+  },
+}
+
+// The reportRequestResponse will contain the report id to use in subsequent requests
+reportRequestResponse, err := xrayManager.GenerateVulnerabilitiesReport(reportRequest)
+```
+
+#### Get Vulnerabilities Report Details
+```go
+reportDetails, err := xrayManager.ReportDetails(reportId)
+```
+
+#### Get Vulnerabilities Report Content
+```go
+reportContentRequest := services.ReportContentRequestParams{
```

Comment:
Let's add the following comment above this line:
// The ReportId value is returned as part of the xrayManager.GenerateVulnerabilitiesReport API response.